config/routes.rb

```diff
@@ -88,6 +88,8 @@
 
     resources :roles, path: 'users', only: [:index, :create, :destroy] do
       collection do
+        get :invite
```

Comment:
This interaction is triggered from an `Invite` link in a form (which is why it made sense at the time to make it a `GET` request).  Otherwise we'll need to have a button that is only triggerable via ajax to submit a `POST`/`PUT`.

I agree that it doesn't quite feel like a `GET` but the interaction is designed that way (and seems like a lot of trouble to do it otherwise, unless I'm missing something obvious).